src/datanode/src/datanode.rs

```diff
@@ -124,6 +140,21 @@ impl Default for OssConfig {
     }
 }
 
+impl Default for AzblobConfig {
```

Comment:
Thank you for your suggestion, I just followed OssConfig and S3Config impl before. Should I modify them together?
https://github.com/GreptimeTeam/greptimedb/blob/develop/src/datanode/src/datanode.rs#LL98C1-L125C2

Reply:
I try to refactor them just now and know why `OssConfig` and `S3Config` don't derive `Default`, because in the struct definition they use `SecretString` type which don't impl `Default` trait, and I use this type too. So we need to manually impl `Default` for it.

```rust
/// Secret strings
#[cfg_attr(docsrs, doc(cfg(feature = "alloc")))]
pub type SecretString = Secret<String>;

impl DebugSecret for String {}
impl CloneableSecret for String {}

impl FromStr for SecretString {
    type Err = core::convert::Infallible;

    fn from_str(src: &str) -> Result<Self, Self::Err> {
        Ok(SecretString::new(src.to_string()))
    }
}
```

Reply:
Got it, thank you.